app/code/Magento/Catalog/Model/Product/Gallery/Processor.php

```diff
@@ -57,25 +57,34 @@ class Processor
      */
     protected $resourceModel;
 
+    /**
+     * @var \Magento\Framework\File\Mime
+     */
+    protected $mime;
```

Comment:
Must be private like any new dependency.

Reply:
fixed